casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -592,17 +592,4 @@ sealed abstract class MultiParentCasperInstances {
         } yield ()
       }
     }
-
-  def fromConfig[
-      F[_]: Monad: Capture: NodeDiscovery: TransportLayer: Log: Time: ErrorHandler: SafetyOracle,
-      G[_]: Monad: Capture: Log: Time](conf: CasperConf, runtimeManager: RuntimeManager)(
-      implicit scheduler: Scheduler): G[MultiParentCasper[F]] =
-    for {
-      genesis <- Genesis.fromInputFiles[G](conf.bondsFile,
-                                           conf.numValidators,
-                                           conf.genesisPath,
-                                           conf.walletsFile,
-                                           runtimeManager)
-      validatorId <- ValidatorIdentity.fromConfig[G](conf)
-    } yield hashSetCasper[F](runtimeManager, validatorId, genesis)
```

Comment:
Double check me here on whether we can actually remove this @birchmd

Reply:
Yeah, if we don't call it anywhere then it is probably fine to get rid of.